.github/workflows/ci.canary.yml

```diff
@@ -217,7 +217,7 @@ jobs:
 
       - name: Build
         run: |
-          yarn set-default-subgraph-release-tag
+          yarn --cwd packages/sdk-core set-default-subgraph-release-tag
```

Comment:
fwiw, https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#defaultsrun also supports a different syntax